okta_openvpn.py

```diff
@@ -151,14 +152,22 @@ def preauth(self):
         path = "/authn"
         data = {
             'username': self.username,
-            'password': self.password,
+            'password': self.password
         }
+
+        if self.device_token_generator == 'UsernameIpGenerator':
+            data['context'] = {
+                'deviceToken': '%s:%s' % (self.username[0:16], self.client_ipaddr)
```

Comment:
I don't know any ISP yet that's using IPv6 but as soon as someone tries to connect using IPv6, this will fail, right? Is it easy to recognise when it happens?

Reply:
Ah, good catch! Didn't think about that. If this overflows, the connection will fail because the Okta API will return an error. Will trim it to [0:16] as well.

Reply:
Adressed in 036d36e.